lib/vite-plugins/vite-plugin-storyblok-bridge.ts

```diff
@@ -0,0 +1,87 @@
+import { generateFinalBridgeObject } from "../utils/generateFinalBridgeObject";
+import { parseAstRawCode } from "../utils/parseAstCode";
+import type { Plugin } from "vite";
+let previousRawCode = [];
```

Comment:
This variable, as well as `rawCode`, `_server`, and `restartTimeout` should be typed more strictly, as they allow for errors right now. Happy to help with this, if needed! 😉